src/model/model.py

```diff
@@ -160,6 +160,105 @@ def apply_operation(self, arr, elements=[], operation='add'):
             arr = function(arr, element, dtype=np.float64)
         return arr
 
+    def apply_formula(self, elements, formula, initialize=True, id=0):
+        """
+        This function calculate complexe operations between float/int and images
+        it handles parenthesis grouping and operator priorities.
+        And it calls "apply_operation" to do pear operations recursively.
+
+        Parameters
+        ----------
+        elements: dict
+            this dictionary contains data (image or float/int...)
+        formula: str
+
```

Comment:
Parameters initialize and id are not listed here, what are they made for?

Reply:
they are useful only for recursivity, I will change the way to manage recursivity to be not confused when calling this method...